### PR TITLE
Packer Integrations

### PR DESCRIPTION
**Pull request by BrandonRomano**
Introduces the Integrations section for Packer.

[🔍 Preview](https://dev-portal-git-brpacker-plugins-hashicorp.vercel.app/packer/integrations)

[📝 Document detailing further migrations](https://whimsical.com/migrating-packer-plugins-F5foVbcDMsaTXJSaWV35ZV)

### CODE REVIEW

File: build-libs/integration-packer-redirects.js [hunk 1]
```diff
@@ -0,0 +1,895 @@
...
+		case TYPE_DATA_SOURCE:
+			return 'datasources'
+		default:
+			return ''
+	}
+}
+
+/**
+ * An array of legacy Packer plugins that were previously in the
+ * Packer Plugin library that are now in the Integrations library.
```
Comment (thiskevinwang) [inline build-libs/integration-packer-redirects.js:25]: If there's a migration guide/document that we can link to in this comment, that would be great.

It's not immediately clear if this list is expected to grow or if it's stable, or it needs to change in tandem with some external work.

**Review by thiskevinwang — APPROVED** [review]

File: build-libs/integration-packer-redirects.js (hunk 1 above)
Comment (zchsh) [reply build-libs/integration-packer-redirects.js:25]: ☝️ related question: this is related to https://github.com/hashicorp/packer/blob/main/website/data/plugins-manifest.json right? Essentially redirecting the plugins detailed in that manifest to their new integration locations?

**Review by zchsh — APPROVED** [review]
✅ Preview looks good to me!

Given we're now redirecting `/packer/plugins` to `/packer/integrations`, should we follow up this PR with one to remove `Plugins` for the Packer nav? (as below)

CleanShot 2023-08-31 at 15 18 27@2x

**Comment by BrandonRomano** [comment]
@zchsh - we're actually going to keep those packer plugin links _for now_ basically until we fully decomission that secion of the website.  I detail it in the video that I talked over this PR, but basically we want to keep those "plugins" links for now so users who are used to navigating to those places get appropriately redirected.

File: build-libs/integration-packer-redirects.js [hunk 2]
```diff
@@ -0,0 +1,895 @@
...
+ *
+ * Redirects are generated from this array.
+ */
```
Comment (BrandonRomano) [inline build-libs/integration-packer-redirects.js:28]:
```suggestion
 * https://whimsical.com/migrating-packer-plugins-F5foVbcDMsaTXJSaWV35ZV
 */
```

**Pull request merged**